os/net/lwip/src/core/mem.c

```diff
@@ -409,6 +409,7 @@ void mem_init(void)
 	ram_end->next = MEM_SIZE_ALIGNED;
 	ram_end->prev = MEM_SIZE_ALIGNED;
 
+	lldbg("[pkbuild] lwip mem range %p %p \n", ram, ram_end);
```

Comment:
could you confirm what "pkbuild" is please?
Seems to be a personal log, in that case could you generalize or remove it please?